api.yaml

```diff
@@ -2,9 +2,9 @@ swagger: '2.0'
 
 # basic meta information
 info:
-  title: Application Lifecycle ManangementAPI
+  title: Application Lifecycle Manangement API
```

Comment:
typo in management. But I would just remove this word anyway, because there is nothing to manage here, just read-only